test/index.html

```diff
@@ -18,6 +18,7 @@
         "../elements/pfe-icon/test/index.html",
         "../elements/pfe-icon-panel/test/index.html",
         "../elements/pfe-number/test/index.html",
+        "../elements/pfe-pagination/test/index.html",
```

Comment:
@kylebuch8 this is causing the automated tests to fail because the pagination component is not in this branch. Is it okay to remove this or at least comment this line out? Thanks!